pyspark/bigdl/keras1/converter.py

```diff
@@ -751,7 +956,7 @@ def create_globalaveragepooling1d(self, klayer, kclayer):
         return seq
 
     def combo_parameter_layer(self, blayer, config):
-        if config["W_constraint"] or config["b_constraint"]:
+        if "W_constraint" in config or "b_constraint" in config:
```

Comment:
revert this back?

Reply:
Do we need to check if config["W_constraint"]=None?

Reply:
DONE